doc/frameworks/symfony.md

```diff
@@ -0,0 +1,481 @@
+                    is_bundle: false
+                translatable:
+                    type: attribute # or annotation
+                    alias: GedmoTranslatable
+                    prefix: Gedmo\Translatable\Document
+                    dir: "%kernel.project_dir%/vendor/gedmo/doctrine-extensions/src/Translatable/Document"
+                    is_bundle: false
+```
+
+To verify your configuration, you can use the `doctrine:mongodb:mapping:info` command to make sure the entities are registered.
+
+```sh
+$ bin/console doctrine:mongodb:mapping:info
+ Found X documents mapped in document manager default:
+ [OK]   Gedmo\Loggable\Document\LogEntry
+ [OK]   Gedmo\Loggable\Document\MappedSuperclass\AbstractLogEntry
+ [OK]   Gedmo\Translatable\Document\MappedSuperclass\AbstractPersonalTranslation
+ [OK]   Gedmo\Translatable\Document\MappedSuperclass\AbstractTranslation
+ [OK]   Gedmo\Translatable\Document\Translation
+```
+
+### ORM Mapping
+
+The below example shows a configuration adding all available mappings to the default entity manager. 
+
+```yaml
+doctrine:
+    orm:
+        default:
+            mappings:
+                loggable:
+                    type: attribute # or annotation
+                    alias: GedmoLoggable
+                    prefix: Gedmo\Loggable\Entity
+                    dir: "%kernel.project_dir%/vendor/gedmo/doctrine-extensions/src/Loggable/Entity"
+                    is_bundle: false
+                translatable:
+                    type: attribute # or annotation
+                    alias: GedmoTranslatable
+                    prefix: Gedmo\Translatable\Entity
+                    dir: "%kernel.project_dir%/vendor/gedmo/doctrine-extensions/src/Translatable/Entity"
+                    is_bundle: false
+                tree:
+                    type: attribute # or annotation
+                    alias: GedmoTree
+                    prefix: Gedmo\Tree\Entity
+                    dir: "%kernel.project_dir%/vendor/gedmo/doctrine-extensions/src/Tree/Entity"
+                    is_bundle: false
+```
+
+To verify your configuration, you can use the `doctrine:mapping:info` command to make sure the entities are registered.
+
+```sh
+$ bin/console doctrine:mapping:info
+ Found X mapped entities:
+ [OK]   Gedmo\Loggable\Entity\LogEntry
+ [OK]   Gedmo\Loggable\Entity\MappedSuperclass\AbstractLogEntry
+ [OK]   Gedmo\Translatable\Entity\MappedSuperclass\AbstractPersonalTranslation
+ [OK]   Gedmo\Translatable\Entity\MappedSuperclass\AbstractTranslation
+ [OK]   Gedmo\Translatable\Entity\Translation
+ [OK]   Gedmo\Tree\Entity\MappedSuperclass\AbstractClosure
+```
+
+## Registering Filters
+
+### Soft Deleteable Filter
+
+When using the [Soft Deleteable](../softdeleteable.md) extension, a filter is available which allows configuring whether
+soft-deleted objects are included in query results.
+
+> [!NOTE]
+> The Symfony bundles default filters to not being enabled, these examples will enable the filters by default.
```

Comment:
The intended message here isn't clear enough to me.
Which bundles are you referring? "doctrine/doctrine-bundle" and "doctrine/mongodb-odm-bundle"?

Reply:
Right.  Basically the `$config->addFilter()` APIs that these configs get transformed into (and same for the Laminas config) only register the filter classes to the object managers, they don't get enabled by default.  The Symfony bundles wire extra calls to enable them globally with their respective `ManagerConfigurator` classes.

Reply:
```suggestion
> The default configuration in the Symfony bundles don't enable the filters. These examples show how to globally enable them.
```

Reply:
Updated, thanks!